{
  "Faults": [
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-02T17:55:00.000+0000",
      "latitude": 51.757714996168595,
      "estimatedRestoration": "2022-03-03T00:01:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JP3153'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JP3153",
      "reference": "JP3153",
      "name": "OX2 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "OX2 6AY",
        "OX2 6AZ"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.267506635788456,
          51.757714996168595
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-03-02T10:15:00.000+0000",
      "UUID": "1be571aa-e8f3-4951-8d08-b99286e9b54d",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.267506635788456,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-02T12:00:00.000+0000",
      "latitude": 50.75667601684237,
      "estimatedRestoration": "2022-03-02T23:59:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JP3528'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JP3528",
      "reference": "JP3528",
      "engineerOnSiteTime": "2022-03-02T15:30:00.000+0000",
      "name": "BH11 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BH11 8NY",
        "BH11 8NZ"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.93077986078874,
          50.75667601684237
        ],
        "type": "Point"
      },
      "UUID": "990a3b8a-1ab9-47a4-bcc7-908d517d0513",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.93077986078874,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-02T18:55:00.000+0000",
      "latitude": 50.78118679842982,
      "estimatedRestoration": "2022-03-03T00:01:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JP3573'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JP3573",
      "reference": "JP3573",
      "engineerOnSiteTime": "2022-03-02T16:30:00.000+0000",
      "name": "PO4 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "PO4 0RD",
        "PO4 0RH",
        "PO4 0RY",
        "PO4 0SA",
        "PO4 0SL",
        "PO4 0SU",
        "PO4 0UD",
        "PO4 0UU",
        "PO5 2EE"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.0797288657057367,
          50.78118679842982
        ],
        "type": "Point"
      },
      "UUID": "c0d8312c-7dec-4de2-a8f7-97d0e049820d",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.0797288657057367,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-02T15:38:00.000+0000",
      "latitude": 51.59384338915403,
      "estimatedRestoration": "2022-03-02T22:25:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JP3572'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JP3572",
      "reference": "JP3572",
      "name": "HP10 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "HP10 0BU",
        "HP10 0BZ"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -0.6806213838952514,
          51.59384338915403
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-03-02T21:00:00.000+0000",
      "UUID": "282c5706-e099-4e6c-87a5-5056e2ab1f65",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.6806213838952514,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-02T18:33:00.000+0000",
      "latitude": 50.73017510529962,
      "estimatedRestoration": "2022-03-03T03:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JP3558'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JP3558",
      "reference": "JP3558",
      "name": "BH1 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BH1 4LD",
        "BH1 4LE",
        "BH1 4LH",
        "BH1 4LQ",
        "BH1 4NF",
        "BH1 4NL"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.8397278308307843,
          50.73017510529962
        ],
        "type": "Point"
      },
      "UUID": "1a34c3a4-a376-4804-9293-6bd7e482cba3",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.8397278308307843,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-02T20:28:00.000+0000",
      "latitude": 51.52283873769813,
      "estimatedRestoration": "2022-03-02T23:40:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JP3737'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JP3737",
      "reference": "JP3737",
      "engineerOnSiteTime": "2022-03-02T21:10:00.000+0000",
      "name": "W5 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "W5 2JN",
        "W5 2JS",
        "W5 2JW",
        "W5 2QP",
        "W5 2QQ",
        "W5 2QT",
        "W5 2QU",
        "W5 2QW",
        "W5 2UB"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -0.30079115890910924,
          51.52283873769813
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-03-02T21:15:00.000+0000",
      "UUID": "2303e6a0-560f-4590-98b5-52130d1f251d",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.30079115890910924,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-02T20:26:00.000+0000",
      "latitude": 51.405255979038955,
      "estimatedRestoration": "2022-03-02T23:45:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JP3734'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JP3734",
      "reference": "JP3734",
      "engineerOnSiteTime": "2022-03-02T21:08:00.000+0000",
      "name": "RG40 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "RG40 2HF",
        "RG40 2HP",
        "RG40 2HS",
        "RG40 2HT",
        "RG40 2HW",
        "RG40 2HY",
        "RG40 2JA",
        "RG40 2JB",
        "RG40 2JD",
        "RG40 2JE"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -0.8226090736402826,
          51.405255979038955
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-03-02T21:30:00.000+0000",
      "UUID": "bcecb05c-af9f-43ad-a869-2e307f2f9bdc",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.8226090736402826,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-02T21:00:00.000+0000",
      "latitude": 51.51166249947143,
      "estimatedRestoration": "2022-03-03T03:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '206315'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/206315",
      "reference": "206315",
      "name": "UB1 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "UB1 3HJ",
        "UB1 3HP"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -0.3705825324039225,
          51.51166249947143
        ],
        "type": "Point"
      },
      "UUID": "9b0e71bd-d5d4-49db-b9f7-821070c6f918",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.3705825324039225,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-02T20:50:00.000+0000",
      "latitude": 51.424511625070004,
      "estimatedRestoration": "2022-03-02T23:59:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JP3754'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JP3754",
      "reference": "JP3754",
      "name": "SN13 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SN13 9AP",
        "SN13 9AR"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -2.1783651839206124,
          51.424511625070004
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-03-02T21:30:00.000+0000",
      "UUID": "0c6f4755-5548-4ebe-a9a7-c1a5fd397dc5",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.1783651839206124,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-02T21:30:00.000+0000",
      "latitude": 51.33268969520819,
      "estimatedRestoration": "2022-03-03T03:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JP3568'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JP3568",
      "reference": "JP3568",
      "engineerOnSiteTime": "2022-03-02T16:55:00.000+0000",
      "name": "GU46 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "GU46 6YF",
        "GU46 6YG",
        "GU46 6YH",
        "GU46 6YJ",
        "GU46 6YL",
        "GU46 6YN",
        "GU46 6YW"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -0.8397127319160135,
          51.33268969520819
        ],
        "type": "Point"
      },
      "UUID": "b2d83403-3f96-438d-8de1-f638f350a358",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.8397127319160135,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-02T21:30:00.000+0000",
      "latitude": 50.904207451860586,
      "estimatedRestoration": "2022-03-03T03:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JP1125'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JP1125",
      "reference": "JP1125",
      "engineerOnSiteTime": "2022-02-27T22:02:00.000+0000",
      "name": "SO19 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SO19 9LB",
        "SO19 9NH",
        "SO19 9NJ",
        "SO19 9NL",
        "SO19 9NR",
        "SO53 1HN"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.3679140603320947,
          50.904207451860586
        ],
        "type": "Point"
      },
      "UUID": "05acf887-6b64-47d4-a068-7e48d0175db3",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.3679140603320947,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-02T21:35:00.000+0000",
      "latitude": 50.95340697311797,
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JP3774'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JP3774",
      "reference": "JP3774",
      "name": "BA21 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BA20 1NR",
        "BA20 1PE",
        "BA21",
        "BA21 3PA",
        "BA21 3PB",
        "BA21 4AF",
        "BA21 4AG",
        "BA21 4AH",
        "BA21 4AJ",
        "BA21 4AL",
        "BA21 4AN",
        "BA21 4AQ",
        "BA21 4AW",
        "BA21 4LT",
        "BA21 4LU",
        "BA21 4NB",
        "BA21 4ND",
        "BA21 4NE",
        "BA21 4NF",
        "BA21 4NG",
        "BA21 4NH",
        "BA21 4NJ",
        "BA21 4NL",
        "BA21 4NN",
        "BA21 4NP",
        "BA21 4NQ",
        "BA21 4NR",
        "BA21 4NS",
        "BA21 4NT",
        "BA21 4NU",
        "BA21 4NW",
        "BA21 4NY",
        "BA21 4NZ",
        "BA21 4PE",
        "BA21 4PF",
        "BA21 4PG",
        "BA21 4PH",
        "BA21 4PJ",
        "BA21 5AR",
        "BA21 5BE",
        "BA21 5BJ",
        "BA21 5BP",
        "BA21 5DR",
        "BA21 5DT",
        "BA21 5DY",
        "BA21 5DZ",
        "BA21 5EB",
        "BA21 5EE",
        "BA21 5EG",
        "BA21 5EH",
        "BA21 5EN",
        "BA21 5ES",
        "BA21 5ET",
        "BA21 5EU",
        "BA21 5EW",
        "BA21 5EX",
        "BA21 5EY",
        "BA21 5FA",
        "BA21 5FD",
        "BA21 5FE",
        "BA21 5FF",
        "BA21 5FG",
        "BA21 5FH",
        "BA21 5FJ",
        "BA21 5FL",
        "BA21 5FN",
        "BA21 5FP",
        "BA21 5FQ",
        "BA21 5FR",
        "BA21 5FU",
        "BA21 5FW",
        "BA21 5FX",
        "BA21 5FZ",
        "BA21 5GA",
        "BA21 5GB",
        "BA21 5GD",
        "BA21 5GE",
        "BA21 5HL",
        "BA21 5HN",
        "BA21 5HR",
        "BA21 5HW",
        "BA21 5JA",
        "BA21 5JB",
        "BA21 5JG",
        "BA21 5JN",
        "BA21 5JQ",
        "BA21 5JX",
        "BA21 5JY",
        "BA21 5JZ",
        "BA21 5PA",
        "BA21 5PB",
        "BA21 5PD",
        "BA21 5PE",
        "BA21 5PF",
        "BA21 5PH",
        "BA21 5PJ",
        "BA21 5PL",
        "BA21 5QA",
        "BA21 5QF",
        "BA21 5QL",
        "BA21 5QP",
        "BA21 5QR",
        "BA21 5RD",
        "BA21 5RE",
        "BA21 5RF",
        "BA21 5RG",
        "BA21 5RH",
        "BA21 5RJ",
        "BA21 5RL",
        "BA21 5RN",
        "BA21 5RQ",
        "BA21 5RR",
        "BA21 5RS",
        "BA21 5RT",
        "BA21 5RU",
        "BA21 5RW",
        "BA21 5RX",
        "BA21 5RY",
        "BA21 5RZ",
        "BA21 5SB",
        "BA21 5SE",
        "BA21 5SF",
        "BA21 5SG",
        "BA21 5SH",
        "BA21 5SQ",
        "BA21 5SR",
        "BA21 5SS",
        "BA21 5ST",
        "BA21 5SU",
        "BA21 5SX",
        "BA21 5SY",
        "BA21 5SZ",
        "BA21 5TN",
        "BA21 5TS",
        "BA21 5UA",
        "BA21 5UB",
        "BA21 5UD",
        "BA21 5UE",
        "BA21 5UF",
        "BA21 5UG",
        "BA21 5UH",
        "BA21 5UJ",
        "BA21 5UL",
        "BA21 5UN",
        "BA21 5UP",
        "BA21 5UQ",
        "BA21 5UR",
        "BA21 5US",
        "BA21 5UW",
        "BA21 5UX",
        "BA21 5UY",
        "BA21 5UZ",
        "BA21 5XA",
        "BA21 5XB",
        "BA21 5XE",
        "BA21 5XF",
        "BA21 5XG",
        "BA21 5XH",
        "BA21 5XJ",
        "BA21 5XL",
        "BA21 5XN",
        "BA21 5XP",
        "BA21 5XQ",
        "BA21 5XR",
        "BA21 5XS",
        "BA21 5XT",
        "BA21 5XU",
        "BA21 5XW",
        "DT3 4BG"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -2.6164639883945107,
          50.95340697311797
        ],
        "type": "Point"
      },
      "UUID": "968c9028-ecb2-4cc7-9875-8581e0e068b4",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.6164639883945107,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-28T09:00:00.000+0000",
      "latitude": 57.48922986542434,
      "estimatedRestoration": "2022-03-05T16:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '205993'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/205993",
      "reference": "205993",
      "name": "HS6 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "HS6 5HX"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -7.221784909692999,
          57.48922986542434
        ],
        "type": "Point"
      },
      "UUID": "c6b32b8f-30a7-44a2-8ae4-a4230efa1d8f",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -7.221784909692999,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-02T17:58:00.000+0000",
      "latitude": 57.5447525125833,
      "estimatedRestoration": "2022-03-02T23:59:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JP3667'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JP3667",
      "reference": "JP3667",
      "name": "AB55 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "AB55 5BH",
        "AB55 5BX",
        "AB55 5DR",
        "AB55 5DX",
        "AB55 5DY",
        "AB55 5DZ",
        "AB55 5EU",
        "AB55 5GD"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -2.9607416247082416,
          57.5447525125833
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-03-02T19:00:00.000+0000",
      "UUID": "2394c206-e2e0-494d-9eed-bd706e736579",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.9607416247082416,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-02T18:07:00.000+0000",
      "latitude": 56.466716667563134,
      "estimatedRestoration": "2022-03-02T23:59:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JP3670'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JP3670",
      "reference": "JP3670",
      "name": "DD5 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "DD5 2DX",
        "DD5 2EX"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -2.868971334720357,
          56.466716667563134
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-03-02T19:00:00.000+0000",
      "UUID": "a84e4ede-6ce4-4d6e-a66f-8f4b06216229",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.868971334720357,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-02T21:36:00.000+0000",
      "latitude": 56.810513723147366,
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JP3772'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JP3772",
      "reference": "JP3772",
      "name": "PH33 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "PH33 6SA",
        "PH33 6SN",
        "PH33 6SP",
        "PH33 6SS",
        "PH33 6SW"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -5.0769553351605,
          56.810513723147366
        ],
        "type": "Point"
      },
      "UUID": "cb33a3af-b534-41e0-b25d-6711d85e47d2",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -5.0769553351605,
      "resolved": false
    }
  ]
}
